jenkins/build_server_win.ps1

```diff
@@ -150,16 +154,21 @@ function Run-UnitTest() {
     Pop-Location
 }
 
-foreach ($arch in @("Win32", "Win64", "ARM")) {
+$ArtifactsDir = "$env:WORKSPACE\artifacts\$($ShaVersion.substring(0, 2))\$ShaVersion"
```

Comment:
does $ArtifactsDir need to be created first or Copy-Item will automatically create it?

Reply:
It is created on the line below this one actually